network/topics/scoring.go

```diff
@@ -249,3 +253,13 @@ func filterCommitteesForTopic(topic string, committees []*storage.Committee) []*
 	}
 	return topicCommittees
 }
+
+// truncateStats returns the subnets in a small format topicNum=ti,fmd,mmd,imd
+func truncateStats(filtered map[string]*pubsub.TopicScoreSnapshot) []string {
+	filteredStrings := make([]string, 0, len(filtered))
+	for topic, snapshot := range filtered {
+		formatted := fmt.Sprintf("%s=%d;%0.3f;%0.3f;%0.3f", topic, snapshot.TimeInMesh, snapshot.FirstMessageDeliveries, snapshot.MeshMessageDeliveries, snapshot.InvalidMessageDeliveries)
+		filteredStrings = append(filteredStrings, formatted)
+	}
+	return filteredStrings
```

Comment:
Instead of returning []string, which would add about 4 characters per item (quotes and commas), we should try to compose one long string separated with just " " in order to take the least amount of characters:

Using `strings.Builder` is the most efficient way to build big strings in Go:
```suggestion
	var b strings.Builder
	i := 0
	for topic, snapshot := range filtered {
		if i > 0 {
			b.WriteString(" ")
		}
		fmt.Fprintf(b, "%s=%d,%0.3f,%0.3f,%0.3f", topic, snapshot.TimeInMesh, snapshot.FirstMessageDeliveries, snapshot.MeshMessageDeliveries, snapshot.InvalidMessageDeliveries)
		i++
	}
	return b.String()
```
(it allocates much less memory than adding strings to slice and later using strings.Join)